resources/views/menus/edit.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('menus.layout')
+
+@section('content')
+    <div class="container">
+        {!! Form::model($menu, ['route' => ['menus.update', 'menus' => $menu->id], 'method' => 'patch' ]) !!}
+
+            @component('menus.create_or_edit_form', [
+                    'listSelectMenu' => $listMenu->except($menu->id)->pluck('name', 'id'),
+                    'parentMenu' => $menu->only('parent_id', 'parent_name'),
+                    'order' => $order,
+                    'listPost' => $listPost->pluck('title', 'id'),
+                    'listPage' => $listPage->pluck('name', 'id'),
+                    'currentPostId' => ($menu->menuable_type == 'App\Models\Post') ? $menu->menuable->id : null,
```

Comment:
làm 1 hàm bên trong để tính toán rồi đẩy ra đây. nếu như này thì khi có 10 type thì em phải có 10 dòng như này à???